atomic_reactor/schemas/config.json

```diff
@@ -453,6 +453,7 @@
                 "expected_media_types": {"$ref": "#/definitions/media_types"}
             },
             "additionalProperties": false,
+            "maxItems": 1,
```

Comment:
Would it be reasonable to add `minItems: 1`? I don't think atomic-reactor can work without registries, at least not in OSBS 2

Reply:
hmm, I agree.